bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -1223,14 +1215,14 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
 
 
 <!-- If an exact match for these grids and start years -->
-<init_interp_attributes sim_year="1979" use_cndv=".false." use_fates=".false." lnd_tuning_mode="clm6_0_cam6.0"
-                        hgrid="0.9x1.25" use_cn=".false." maxpft="17"
->hgrid=0.9x1.25 maxpft=17 mask=gx1v7 use_cn=.false. use_crop=.false. irrigate=.true. glc_nec=10 do_transient_pfts=.false. use_excess_ice=.false.
+<init_interp_attributes sim_year="1979" use_cndv=".false." use_fates=".false." phys="clm6_0"
+                        hgrid="0.9x1.25"
+>maxpft=79 mask=gx1v7 use_cn=.true. use_crop=.true. irrigate=.true. glc_nec=10 use_excess_ice=.true.
 </init_interp_attributes>
 
-<init_interp_attributes sim_year="1979" use_cndv=".false." use_fates=".false." lnd_tuning_mode="clm6_0_cam6.0"
-                        hgrid="1.9x2.5" use_cn=".false." maxpft="17"
->hgrid=1.9x2.5 maxpft=17 mask=gx1v7 use_cn=.false. use_crop=.false. irrigate=.true. glc_nec=10 do_transient_pfts=.false. use_excess_ice=.false.
+<init_interp_attributes sim_year="1979" use_cndv=".false." use_fates=".false." phys="clm6_0"
+                        hgrid="1.9x2.5"
+>hgrid=0.9x1.25 maxpft=79 mask=gx1v7 use_cn=.true. use_crop=.true. irrigate=.true. glc_nec=10 use_excess_ice=.true.
```

Comment:
I appreciate I'm joining the conversation at the tail end of this PR, but wanted to note we're replacing a 16 pft SP case 78 pft BGC-Crop case here.  Is this intended?

Reply:
@wwieder as you said in another comment, if this happens to be wrong for 1979, let's address it at a later time. For now though, Erik and I looked at these settings at length, and now they work for the manual tests that we tried.